server/index.js

```diff
@@ -1,10 +1,105 @@
 // Entry point of express server
 const express = require('express');
+const cors = require('cors');
+const { Pictures, ReviewPhotos } = require('../mongoDB/index');
+
 const app = express();
 const port = 3000;
```

Comment:
consider changing this to `const port = process.env.PORT || 3000;`  That way if you go to deploy it you'll be ready for whatever port Heroku (or whoever) wants to put you on.